tests/integration/compose/docker_compose_vault.yml

```diff
@@ -0,0 +1,10 @@
+services:
+    openbao:
+        image: openbao/openbao:2.3
+        restart: always
+        environment:
+            BAO_DEV_ROOT_TOKEN_ID: foobar
+            BAO_DEV_LISTEN_ADDRESS: "0.0.0.0:1337"
+            BAO_ADDR: "http://0.0.0.0:1337"
+        ports:
```

Comment:
Hardcoded port numbers is a source of problems.
Not sure, but if it is possible to use 'expose' instead of 'ports', it is great.

Reply:
Replaced it with default 8200 port. Also "ports" are better for dev tests. While "expose" is better to productions. So I keep usage of "ports".